Komodo/Assets/Packages/KomodoCore/Runtime/Scripts/RuntimeSession/KomodoMenu.cs

```diff
@@ -25,8 +25,6 @@ public class KomodoMenu : MonoBehaviour
 
         public GameObject startCapture;
```

Comment:
TODO: commit this somewhere for safekeeping and then remove the unneeded reference